#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 41687eab1c3212037a5a0a847407381e8a5c69bd Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Fri, 15 Mar 2024 08:49:49 +0100
Subject: [PATCH] WIP: Find moc from host installation

This is just a workaround until we build host tools
in Gpg4win
---
 m4/qt6.m4 | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/m4/qt6.m4 b/m4/qt6.m4
index 8a3322a7..c0dbcc3d 100644
--- a/m4/qt6.m4
+++ b/m4/qt6.m4
@@ -27,6 +27,11 @@ AC_DEFUN([FIND_QT6],
     # Qt6 moved moc to libexec
     qt6libexecdir=$($PKG_CONFIG --variable=libexecdir 'Qt6Core >= 6.4.0')
     AC_PATH_TOOL(MOC, moc, [], [$qt6libexecdir])
+    if test -z "$MOC"; then
+      if test "$have_w32_system" = "yes"; then
+        AC_PATH_TOOL(MOC, moc, [], [/usr/libexec/qt6])
+      fi
+    fi
     if test -z "$MOC"; then
       AC_MSG_WARN([moc not found - Qt 6 binding will not be built.])
       have_qt6_libs="no";
--
2.43.2
